acceptance/selftest/record_cloud/workspace-file-io/script

```diff
@@ -0,0 +1,62 @@
+export username=$($CLI current-user me | jq -r .userName)
+
+# MSYS2 automatically converts absolute paths like /Users/$username/$UNIQUE_NAME to
+# C:/Program Files/Git/Users/$username/UNIQUE_NAME before passing it to the CLI
+# Setting this environment variable prevents that conversion on windows.
+export MSYS_NO_PATHCONV=1
```

Comment:
Is this ever behavior we actually want? If not, this can be set in the top-level configuration file.

Reply:
Good point, addressed in https://github.com/databricks/cli/pull/2851

Reply:
There's a bunch of tests that started failing because of setting this globally. For now the ROI does not seem worth it to me. I'm closing the PR without following up on this thread for now.

Reply:
Thanks for taking a look.